use register_cmu_us_kal instead of register_cmu_us_kal16

Index: Tools/speech/GNUmakefile
--- Tools/speech/GNUmakefile.orig
+++ Tools/speech/GNUmakefile
@@ -21,8 +21,8 @@ GSSpeechServer_OBJC_FILES = \
 
 
 ifeq ($(GSSPEECHENGINE), Flite)
-FLITE_BASE_LIBS = -lflite_usenglish -lflite_cmulex -lflite
-FLITE_EXTRA_LIBS = -lflite_cmu_us_kal16
+FLITE_BASE_LIBS = -lflite_usenglish -lflite_cmulex -lflite -lsndio
+FLITE_EXTRA_LIBS = -lflite_cmu_us_kal
 
 GSSpeechServer_OBJC_FILES += FliteSpeechEngine.m
 GSSpeechServer_INCLUDE_DIRS += -I/usr/local/include/flite -I../../Headers \
